### PR TITLE
Update/3 add pact bytecode

### LINKED ISSUE #3: Update codec implementation to embed pact bytecode
relates to https://github.com/cennznet/cennznut-rs/issues/3

### PR DESCRIPTION
**Pull request by elim0322**
closes #3

### CODE REVIEW

File: src/cennznut/versions/0/index.js [hunk 1]
```diff
@@ -38,6 +41,29 @@ function normaliseBlockCooldown(
   }
 }
 
+function verifyConstraints(method, moduleName, methodName) {
+  if (method.constraints) {
+    if (!Array.isArray(method.constraints)) {
+      throw new Error(
+        `Module "${moduleName}"'s method "${methodName}" ` +
+        `has invalid constraints specified`
```
Comment (holygits) [inline src/cennznut/versions/0/index.js]: Can update this error message to make clear than an `Array` was expected

File: src/cennznut/versions/0/index.js [hunk 2]
```diff
@@ -38,6 +41,29 @@ function normaliseBlockCooldown(
...
+    if (method.constraints.length == 0) {
+      throw new Error(
+        `Module "${moduleName}"'s method "${methodName}" ` +
+        `has no constraints specified`
+      )
+    }
+    if (method.constraints.length > MAX_CONSTRAINTS_BYTE_LENGTH) {
+      throw new Error(
+        `Module "${moduleName}"'s method "${methodName}" ` +
+        `has more constraints than the allowed ${MAX_CONSTRAINTS_BYTE_LENGTH}`
```
Comment (holygits) [inline src/cennznut/versions/0/index.js]: "constraints length must be <= ${MAX_CONSTRAINTS_BYTE_LENGTH}"

**Comment by holygits** [comment]
LGTM. Don't forget to rebase incase you merge #7 first

**Comment by wilfred-centrality** [comment]
@elim0322 please let me know where and how the above are being addressed

**Pull request merged**